backend/src/test/java/gov/cdc/usds/simplereport/test_util/SliceTestConfiguration.java

```diff
@@ -58,22 +54,10 @@
 
   @Bean
   public IdentitySupplier testIdentityProvider() {
-    final Map<String, IdentityAttributes> identityLookup =
-        Stream.of(
-                TestUserIdentities.STANDARD_USER_ATTRIBUTES,
-                TestUserIdentities.SITE_ADMIN_USER_ATTRIBUTES)
-            .collect(Collectors.toMap(IdentityAttributes::getUsername, i -> i));
-    return () -> {
-      Authentication auth = SecurityContextHolder.getContext().getAuthentication();
-      if (auth == null) {
-        // needed for running the commandlinerunner that we don't actually need
-        return TestUserIdentities.STANDARD_USER_ATTRIBUTES;
-      }
-      String username = auth.getName();
-      LoggerFactory.getLogger(SliceTestConfiguration.class)
-          .warn("Found username {} for auth {}", username, auth);
-      return identityLookup.get(username);
-    };
+    return new DemoUserIdentitySupplier(
+        List.of(
+            new DemoUser(null, TestUserIdentities.STANDARD_USER_ATTRIBUTES),
```

Comment:
i'm confused why we're setting their org role claims to null? will this be a problem?

Reply:
It will not—this is an IdentitySupplier implementation, so it doesn't care about roles. In the slice context, we're providing the roles as authorities that are hard-coded into the annotations later on in this file, which this PR doesn't change.

Reply:
can you add a quick comment stating the relevant parts of this?